### ISSUE TITLE
Question on grid cell weights

### ISSUE THREAD

**Issue by atmos-project** [andersy005/xarray-tutorial#5]
Hi Anderson,

Thank you very much for this tutorial. It will going to help me a lot since I need to deal with netcdf files from climate models.

I have a basic question. It is not exactly about python but about the grid cell weights you used in your notebook #4: computation. I did not understand quite well in which situations we should use these weights. By the way, are the weights simply the grid cell area? Is there any reference (website/text) that you can indicate for those who need to process gridded data and not to have these type of errors?

Thanks

**Comment by andersy005** [comment]
@atmos-project, 

> I have a basic question. It is not exactly about python but about the grid cell weights you used in your notebook #4: computation. I did not understand quite well in which situations we should use these weights. . By the way, are the weights simply the grid cell area? Is there any reference (website/text) that you can indicate for those who need to process gridded data and not to have these type of errors?

Yes, the weights in that example are just `grid cell` area. Another example in which you may want to use weights is when let's say you have daily data and you want to compute monthly averages. In this case, you may want to make sure that you weigh each month accordingly (since months have different number of days). So, in this case, the weights would correspond to the number of days in each month.

For references, take a look at the cf-conventions, more specifically the cell-measures section: http://cfconventions.org/cf-conventions/cf-conventions.html#cell-measures

I hope this helps.

**Comment by atmos-project** [comment]
Right, Thanks for the reply.

By the way, your example of computing monthly averages from daily outputs is exactly what I will do. I have a dataset with daily outputs but I need monthly averages.  I thought about using Xarray and groupby. I thought that I would not have to worry about the different number of days depending on the month using  groupby. Am I wrong? I mean, I thought Xarray/groupby would handle this.

Another quick question, do you know how I compute the gridcell area? My grid is a regular 0.5 deg. res.

Thanks.
